src/test/java/com/wikia/webdriver/common/dataprovider/mobile/MobileAdsDataProvider.java

```diff
@@ -57,8 +57,6 @@ private MobileAdsDataProvider() {
             "SyntheticTests/DfpParams",
             "wka.ent/_adtest//article",
             "MOBILE_TOP_LEADERBOARD",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "MOBILE_TOP_LEADERBOARD" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)